components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultLogoutRequestHandler.java

```diff
@@ -574,8 +576,8 @@ private String getDefaultLogoutReturnUrl() {
         String defaultLogoutUrl = IdentityUtil.getProperty(DEFAULT_LOGOUT_URL_CONFIG);
         if (StringUtils.isBlank(defaultLogoutUrl)) {
             if (log.isDebugEnabled()) {
-                log.debug("The default logout URL is not set in the identity.xml file. Therefore directing to the " +
-                        "default logout page of the server.");
+                log.debug("The default logout URL is not set in the deployment.toml file. Therefore directing to" +
```

Comment:
better ti keep it as identity.xml because all configs will be set through deployment.toml but it will get affect the identity.xml file for this case.

Reply:
Fixed.